src/chainparams.cpp

```diff
@@ -390,11 +390,11 @@ class CRegTestParams : public CChainParams {
         consensus.signet_challenge.clear();
         consensus.nSubsidyHalvingInterval = 150;
         consensus.BIP16Exception = uint256();
-        consensus.BIP34Height = 2; // BIP34 activated on regtest (Block at height 1 not enforced for testing purposes)
+        consensus.BIP34Height = 1; // Always active unless overridden
         consensus.BIP34Hash = uint256();
-        consensus.BIP65Height = 111; // BIP65 activated on regtest (Block at height 110 and earlier not enforced for testing purposes)
-        consensus.BIP66Height = 102; // BIP66 activated on regtest (Block at height 101 and earlier not enforced for testing purposes)
-        consensus.CSVHeight = 432; // CSV activated on regtest (Used in rpc activation tests)
+        consensus.BIP65Height = 1;  // Always active unless overridden
+        consensus.BIP66Height = 1;  // Always active unless overridden
+        consensus.CSVHeight = 1;    // Always active unless overridden
         consensus.SegwitHeight = 0; // SEGWIT is always activated on regtest unless overridden
```

Comment:
nit: For consistency reasons, could also set `SegwitHeight` to 1? Otherwise this implies to the reader that there's a subtle difference. (Maybe there is one that I'm not aware of, but at least the tests still seem to pass with a value of 1).

Reply:
Thanks, done.